operator/controllers/common/phasehandler.go

```diff
@@ -79,6 +81,11 @@ func (r PhaseHandler) HandlePhase(ctx context.Context, ctxTrace context.Context,
 			piWrapper.Complete()
 			piWrapper.SetState(common.StateFailed)
 			spanTrace.AddEvent(phase.LongName + " has failed")
+			if msg, err := r.createFailureReasonSpanEvents(ctx, phase, piWrapper); err != nil {
+				r.Log.Error(err, "cannot create failure spans")
```

Comment:
I don't think this is worth to generate an error. It is more of a warning IMHO. Furthermore, spans are created and exported since the `.End()` method will be called regardless of this error
```suggestion
				r.Log.Warning(err, "cannot add failure events to spans")
```

Reply:
we do not have a .Warn() method here, only .Info() or .Error()